ex8_ch3.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input
+number_people = int(input("Enter the number of people: "))
+total_pizzas = int(input("Enter the total number of pizzas: "))
+slices_per_pizza = int(input("Enter the slices per pizza: "))
+
+#step2: calculate the total slices of pizza
+total_slices = total_pizzas * slices_per_pizza
+
+#step3: calucate the slices per person
+slices_per_person = total_slices // number_people
```

Comment:
I think you want floor division. This divides the pieces into whole slices. It doesn't look like the problem is concerned with the it being fractional, like 2.5.

Reply:
However, it looks like the problem wants you to print out the remaining pieces as well as _whole_ pieces_ per person. So in this case you can use the modulo operator to find that.